src/Tools/dotnet-trace/CommandLine/Commands/ListProfilesCommandHandler.cs

```diff
@@ -65,19 +64,29 @@ public static async Task<int> GetProfiles(IConsole console)
                                   (ulong)ClrTraceEventParser.Keywords.Exception,
                         eventLevel: EventLevel.Verbose),
                 },
-                "Tracks allocation and collection performance."),
+                "Tracks allocation and collection performance at the most verbose level."),
+            new Profile(
+                "gc-verbose",
+                new Provider[] {
+                    new Provider(
+                        name: "Microsoft-Windows-DotNETRuntime",
+                        keywords: (ulong)ClrTraceEventParser.Keywords.GC |
+                                  (ulong)ClrTraceEventParser.Keywords.GCHandle |
+                                  (ulong)ClrTraceEventParser.Keywords.Exception,
+                        eventLevel: EventLevel.Verbose
+                    ),
+                },
+                "Tracks GC and GC handle events at a more verbose level"),
```

Comment:
Lets update this description to indicate that object allocations are sampled in addition to GC collection info.